src/hotspot/share/gc/shenandoah/shenandoahPhaseTimings.hpp

```diff
@@ -99,8 +102,8 @@ class outputStream;
   f(conc_rendezvous_roots,                          "Rendezvous")                      \
   f(conc_evac,                                      "Concurrent Evacuation")           \
                                                                                        \
-  f(init_update_refs_gross,                         "Pause Init  Update Refs (G)")      \
-  f(init_update_refs,                               "Pause Init  Update Refs (N)")      \
+  f(init_update_refs_gross,                         "Pause Init Update Refs (G)")      \
+  f(init_update_refs,                               "Pause Init Update Refs (N)")      \
```

Comment:
Revert this change, we should clean this up upstream.

Reply:
Created a pull request for this: https://github.com/openjdk/jdk/pull/3342